includes/snmp.inc.php

```diff
@@ -264,6 +282,29 @@ function snmp_get($device, $oid, $options = null, $mib = null, $mibdir = null)
 }//end snmp_get()
 
 
+function snmp_get_next($device, $oid, $options = null, $mib = null, $mibdir = null)
+{
+    global $runtime_stats;
+
+    if (strstr($oid, ' ')) {
+        echo report_this_text("snmp_get_next called for multiple OIDs: $oid");
```

Comment:
I've also changed this echo statement for the snmp_get function I based mine on.